pkg/acm/policy.go

```diff
@@ -0,0 +1,239 @@
+package acm
```

Comment:
I think it would be better to have this called `ocm` because as it seems it uses open-cluster-management.io